__tests__/components/Comments/Comment.test.js

```diff
@@ -4,6 +4,7 @@ import {fireEvent, render} from '@testing-library/react-native';
 import * as UserUtil from '../../../src/utils/users';
 import * as serviceVote from '../../../src/service/vote';
 import Comment, {isEqual} from '../../../src/components/Comments/Comment';
+import Store from '../../../src/context/Store';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `Store` from `'../../../src/context/Store'` has been added. Ensure that the `Store` context provides all necessary values and functions required by the `Comment` component. If not, it might lead to unexpected behavior or crashes.
```diff
+ import Store from '../../../src/context/Store';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->